Libs/MRML/Logic/vtkMRMLColorLogic.h

```diff
@@ -39,6 +39,57 @@ class vtkMRMLColorTableNode;
 class VTK_MRML_LOGIC_EXPORT vtkMRMLColorLogic : public vtkMRMLAbstractLogic
 {
 public:
+  typedef struct
+    {
+      std::string CodeValue;
+      std::string CodingSchemeDesignator;
+      std::string CodeMeaning;
+
+      void StandardTerm() {
+        CodeValue = "";
+        CodingSchemeDesignator = "";
+        CodeMeaning = "";
+      }
+      void StandardTerm(std::string value, std::string designator, std::string meaning) {
+        CodeValue = value;
+        CodingSchemeDesignator = designator;
+        CodeMeaning = meaning;
+      }
+
+      void PrintSelf(ostream &os) {
```

Comment:
May be you could use `vtkIndent` ? and also implement the `<<` operator for these struct.